OMEdit/OMEditLIB/OMC/OMCProxy.cpp

```diff
@@ -1535,14 +1535,11 @@ QString OMCProxy::getDocumentationAnnotation(LibraryTreeItem *pLibraryTreeItem)
   QList<QString> docsList = mpOMCInterface->getDocumentationAnnotation(pLibraryTreeItem->getNameStructure());
   QString infoHeader = "";
   infoHeader = getDocumentationAnnotationInfoHeader(pLibraryTreeItem->parent(), infoHeader);
-  QString doc = "";
-  // get the model name and show it as the first line on the documentation page.
-  doc = "<h2>" % pLibraryTreeItem->getNameStructure() % "</h2>" ;
-
-  // get the model description if available (e.g) model a "test sample" end a;
-  QString description = getClassComment(pLibraryTreeItem->getNameStructure());
-  if (!doc.isEmpty()) {
-    doc = doc % " <p> <i>" % description % "</i> </p>";
+  QString doc = "<h2>" % pLibraryTreeItem->getNameStructure() % "</h2>";
+  // get the class comment if available e.g., model a "test sample" end a;
+  QString comment = getClassComment(pLibraryTreeItem->getNameStructure());
+  if (!comment.isEmpty()) {
+    doc = doc % "<h4>" % comment % "</h4>";
```

Comment:
It should not be header, It is model description text and this should be 
`doc = doc % " <p> <i>" % description % "</i> </p>";` see here https://github.com/OpenModelica/OpenModelica/issues/12465#issuecomment-2162260020

Reply:
It was a regression from https://github.com/OpenModelica/OpenModelica/commit/502675a2a1dbd3fb110b3bd2becf8ca4f942f6aa. The class comment was overwritten when I changed to use `QStringBuilder`. The user just noticed now that the class comment is not shown and suggested to show it in italic.